genesyscloud/routing_wrapupcode/data_source_genesyscloud_routing_wrapupcode.go

```diff
@@ -0,0 +1,37 @@
+package routing_wrapupcode
+
+import (
+	"context"
+	"fmt"
+	"terraform-provider-genesyscloud/genesyscloud/provider"
+	"terraform-provider-genesyscloud/genesyscloud/util"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/retry"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func dataSourceRoutingWrapupcodeRead(ctx context.Context, d *schema.ResourceData, m interface{}) diag.Diagnostics {
+
+	sdkConfig := m.(*provider.ProviderMeta).ClientConfig
+	proxy := newRoutingWrapupcodeProxy(sdkConfig)
```

Comment:
`newRoutingWrapupcodeProxy` -> `getRoutingWrapupcodeProxy`

`newRoutingWrapupcodeProxy` should only be called from inside `getRoutingWrapupcodeProxy`

Reply:
Thanks for catching this :) very bad coding